signer/pkcs11/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package pkcs11 implements support for PKCS #11 signers. If the
+// package has not been built with the `pkcs11` tag, the `New`
+// function will be a stub.
+package pkcs11
```

Comment:
This package might want to be at the top level, rather than under signer.  We're going to want to use PKCS#11 for OCSP signing as well.   So I guess the choice is whether you want the dependencies to go:
1. ocsp -> signer/pkcs11
2. ocsp -> pkcs11